performance_tests/README.md

```diff
@@ -0,0 +1,47 @@
+# Performance tests
+
+This readme is to show the reader how to run a .jmx file to display our sites JMeter performance tests results.
+
+## Explanation of JMeter
+The Apache JMeter application is open source software, a 100% pure Java application designed to load test functional behavior and measure performance. It was originally designed for testing Web Applications but has since expanded to other test functions.
+
+Apache JMeter may be used to test performance both on static and dynamic resources, Web dynamic applications.
+It can be used to simulate a heavy load on a server, group of servers, network or object to test its strength or to analyse overall performance under different load types.
+
+## JMeter download and usage
+
+You can download JMeter from the [official page](https://jmeter.apache.org/). After installation, you can use your terminal to run the JMeter.sh file to do this,  navigate to the bin directory and run the command: 
+
+```
+    sh jmeter.sh
+```
+
+Your JMeter window will open. You can click the open icon and navigate to this directory 'performance tests' you will find different tests types which contain their .csv results files and .jmx files.
+
+
+Selecting one of the .jmx files will open up the details of the test in your JMeter window.
+
+You can amend the test or run it using the play icon. Note if you want to run a test, under 'User Defined Variables' you will need to specify the number of users in the test, your staff name for the path e.g. john_smith, ramp up time and test number.
```

Comment:
staff_name ? Is this your username or something else? 
Can you give the example you use and why it is needed? (e.g does it determine where results are written or something?)

If you have time it would be good to show an example of what you need to do/change if say you are running:
scenario: load
users: 300
duration: X minutes

Reply:
Sure thing I need to remove staff name as JMeter cannot handle it but I will add more details now.